ibis/backends/bigquery/tests/test_compiler.py

```diff
@@ -152,7 +152,7 @@ def test_day_of_week(case, expected, dtype, strftime_func):
     )
 
     expr_name = date_var.day_of_week.full_name()
-    result = ibis.bigquery.compile(expr_name)
+    result = ibis.backends.bigquery.compile(expr_name)
```

Comment:
```suggestion
    result = ibis.bigquery.compile(expr_name)
```
In this one and all the rest, this is not the `ibis.bigquery`/`ibis.backends.bigquery` module. Instead, it's the attribute `bigquery` of ibis (in `ibis/__init__.py`). Which is that module actually, but created as an attribute of the module ibis. Very confusing, I know, but luckily, with this PR there will be no ambiguity anymore (that's one of the reasons for the move of the backend).

Reply:
Great! I get it now!